core/src/main/java/com/arcrobotics/ftclib/hardware/motors/MotorGroup.java

```diff
@@ -40,6 +48,113 @@ public double get() {
         return group[0].get();
     }
 
+    /**
+     * @return All motor target speeds as a percentage of output
+     */
+    public List<Double> getSpeeds() {
+        return Arrays.stream(group)
+                .map(Motor::get)
+                .collect(Collectors.toList());
+    }
+
+    @Override
+    public double getVelocity() {
+        return group[0].getCorrectedVelocity();
+    }
+
+    /**
+     * @return All current velocities of the motors in the group
+     */
+    public List<Double> getVelocities() {
+        return Arrays.stream(group)
+                .map(Motor::getCorrectedVelocity)
+                .collect(Collectors.toList());
+    }
+
+    @NotNull
+    @Override
+    public Iterator<Motor> iterator() {
+        return Arrays.asList(group).iterator();
+    }
+
+    @Override
+    public Encoder setDistancePerPulse(double distancePerPulse) {
+        Encoder leaderEncoder = group[0].setDistancePerPulse(distancePerPulse);
+        for (int i = 1; i < group.length; i++) {
+            group[i].setDistancePerPulse(distancePerPulse);
+        }
+        return leaderEncoder;
+    }
+
+    /**
+     * @return The position of every motor in the group in units of distance
+     * which is by default ticks
+     */
+    public List<Double> getPositions() {
+        return Arrays.stream(group)
+                .map(Motor::getDistance)
+                .collect(Collectors.toList());
+    }
+
+    @Override
+    public void setRunMode(RunMode runmode) {
+        group[0].setRunMode(runmode);
+    }
+
+    @Override
+    public void setZeroPowerBehavior(ZeroPowerBehavior behavior) {
+        for (Motor x : group) {
+            x.setZeroPowerBehavior(behavior);
```

Comment:
```suggestion
        for (Motor motor : group) {
            motor.setZeroPowerBehavior(behavior);
```
I think we can live with the extra 4 bytes from using `motor` instead of `x` :upside_down_face:.